src/standalone/Vegetation/photosynthesis.jl

```diff
@@ -1,4 +1,33 @@
-export FarquharParameters, FarquharModel, C3, C4
+export SIFParameters, FarquharParameters, FarquharModel, C3, C4
+
+abstract type AbstractPhotosynthesisModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    SIFParameters{FT<:AbstractFloat}
+
+The required parameters for the SIF parameterisation [give citation].
+$(DocStringExtensions.FIELDS)
+"""
+@kwdef struct SIFParameters{FT <: AbstractFloat}
+    ""
+    kf::FT = FT(0.05)
+    ""
+    kd_p1::FT = FT(0.03)
+    ""
+    kd_p2::FT = FT(0.0273)
+    ""
+    min_kd::FT = FT(0.087)
+    ""
+    kn_p1::FT = FT(6.2473)
+    ""
+    kn_p2::FT = FT(0.5944)
+    ""
+    kp::FT = FT(4.0)
+    ""
+    kappa_p1::FT = FT(0.045)
+    ""
+    kappa_p2::FT = FT(7.85)
```

Comment:
Intercept of line relating leaf-level fluorescence to spectrometer-observed fluorescence as a function of Vcmax 25. Obtained from Figure 1 of Lee et al 2015.